service/debugger/debugger.go

```diff
@@ -149,7 +150,18 @@ func (d *Debugger) CreateBreakpoint(requestedBp *api.Breakpoint) (*api.Breakpoin
 	)
 	switch {
 	case len(requestedBp.File) > 0:
-		addr, err = d.process.FindFileLocation(normalizePath(requestedBp.File), requestedBp.Line)
+		fileName := requestedBp.File
+		if runtime.GOOS == "windows" {
```

Comment:
Instead of adding the check here, let's remove the assumption in `FindFileLocation` that it's input has been normalized, let's take that burden off the caller and centralize that logic within that method. That way we're not special casing throughout the codebase.

Additionally, since `normalizePath` already is checking `runtime.GOOS` let's remove that from any logic that calls it. I'm also unsure about the iteration here, we break when `fileNameNormalized == normalizePath(symfile)`, but why can't we just assume that the string returned from `normalizePath` will correctly match a file (if it exists in sources)? Since, if that iteration fails to find anything `fileName` will not be normalized at all, and we don't bail if we can't find anything either, so that check isn't buying us anything it seems.